gpuflow/dataplane/gpu/cuda/cuda_lpm_factory.cu

```diff
@@ -147,18 +185,49 @@ int IPv6LPMFactory::AddLPMRule(uint8_t *ipv6_address, uint8_t depth, uint8_t nex
     start = right_shift << (24 - depth);
       
   } else {
-    // TODO: Handling when depth is larger than 24
+    // Handling when depth is larger than 24
     start = ipv6_address[0] << 16 | ipv6_address[1] << 8 | ipv6_address[2];
+   
+    // Calculate the number of tbl8 table 
+    int tbl_8_number = depth % 8 ? ((depth - 24) / 8) + 1 : (depth - 24) / 8;
```

Comment:
Looks not correct, consider depth = 25?

Reply:
When depth = 25, there would be 1 tbl8 table?

Reply:
But `(25-24) / 8 == 0`?

Reply:
The correct version is 
```c++
int num_of_tbl8 = (depth - 24) / 8 + 1;
```